File: ProyectosCSharp/SoftProg/SoftProgBusiness/Almacen/BO/IProductoBO.cs
using SoftProgBusiness.BO;
using SoftProgModel.Almacen;
using SoftProgModel.GestClientes;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftProgBusiness.Almacen.BO
{
    public interface IProductoBO : IBaseBO<Producto>
    {
        BindingList<Producto> listarPorNombre(string nombre);
    }
}

File: BackJava_FrontCSharp/BackJava/SoftProgBusiness/src/main/java/pe/edu/pucp/softprog/almacen/boi/IProductoBO.java
package pe.edu.pucp.softprog.almacen.boi;

import java.util.ArrayList;
import pe.edu.pucp.softprog.almacen.model.Producto;
import pe.edu.pucp.softprog.bo.IBaseBO;

public interface IProductoBO extends IBaseBO<Producto>{
    ArrayList<Producto> listarPorNombre(String nombre);
}
